host/3.0/publish.yml

```diff
@@ -157,11 +158,15 @@ steps:
       docker pull $SOURCE_REGISTRY/node:$(PrivateVersion)-node14-slim
       docker pull $SOURCE_REGISTRY/node:$(PrivateVersion)-node14-appservice
 
+      docker tag $SOURCE_REGISTRY/node:$(PrivateVersion)-node8-appservice $TARGET_REGISTRY/node:$(TargetVersion)-appservice
```

Comment:
This line should be removed. It looks overwrite current target version. current node:3.0 will be node10. 
https://github.com/Azure/azure-functions-docker